### ISSUE TITLE
Post merger clean up

### ISSUE THREAD

**Issue by Sepitus-exe** [Sepitus-exe/WKElementaryDark#23]
Dashboard progress bar text set to lightbg variant, when on darkbg|
![image](https://user-images.githubusercontent.com/82216846/212811558-c4d3f4de-b708-487a-b3a1-8ff805d7ecf4.png)
FIXED
![image](https://user-images.githubusercontent.com/82216846/213038949-10301719-626f-46f0-9617-61eef7e35a8e.png)

**Comment by Sepitus-exe** [comment]
Empty kotoba boxes are unstiled
![image](https://user-images.githubusercontent.com/82216846/212811630-fa442a20-012b-4706-a2df-c6f7c488c07b.png)

**Comment by Sepitus-exe** [comment]
progress bars in levels/radical/kanji/vocab have an escapee green box-shadow + probably a wrong bar color
![image](https://user-images.githubusercontent.com/82216846/212811857-f00c636e-0d65-4a29-9073-56c1c701ced7.png)

**Comment by Sepitus-exe** [comment]
0 amount header buttons look wrong (3 sided borders should not happen)
![image](https://user-images.githubusercontent.com/82216846/212812005-6e091d74-eba2-4781-8687-160bf61eb10b.png)

**Comment by Sepitus-exe** [comment]
Some left side padding should be added to the header of individual item pages
![image](https://user-images.githubusercontent.com/82216846/212812337-dbdf7a71-0f80-466a-a801-b24e630f35b6.png)

**Comment by Sepitus-exe** [comment]
super minor, but :active buttons in reviews have different background under characters than what the rest of the button uses

![image](https://user-images.githubusercontent.com/82216846/212812456-d900a345-eb65-45cd-9553-6885edc17c41.png)

**Comment by Sepitus-exe** [comment]
Kanji styling is bleeding into higher elements in reviews
![image](https://user-images.githubusercontent.com/82216846/212812874-22f58e37-2a29-4c27-9d2a-64341910c642.png)
This one is probably a carry over from the original file, not introduced with the new changes.

**Comment by Sepitus-exe** [comment]
R/L buttons dont change background colors, depending on if they are empty or not
![image](https://user-images.githubusercontent.com/82216846/212813137-b4dcc67e-0e53-4fcf-9e79-8c9458282a99.png)

**Comment by Sepitus-exe** [comment]
Escapee chartreuse
![image](https://user-images.githubusercontent.com/82216846/212813396-0e547521-d51a-4fe5-bb0e-1933de1d02e9.png)

**Comment by Sepitus-exe** [comment]
Distorted lesson quiz header (intentional?)
![image](https://user-images.githubusercontent.com/82216846/212813544-488e2721-6946-41eb-8f57-d637efb62074.png)

**Comment by wrex** [comment]
Awesome. Nice bite-sized things to tackle. These are all **EXACTLY** the kinds of things I hoped to find and start tackling.

It's probably better going forward to open separate issues and assigning them, even for little things like this. Otherwise we risk stepping on each other as we both work on the same things.

> Dashboard progress bar text set to lightbg variant, when on darkbg| ![image](https://user-images.githubusercontent.com/82216846/212811558-c4d3f4de-b708-487a-b3a1-8ff805d7ecf4.png)

This one caught me by surprise. I didn't even realize the text stays in that position. I thought it was positioned over the bar. On my screen it's over the yellow.

The are all great, though. Some of these things I didn't even realize existed. More eyes are extremely helpful. 

I'm especially glad the chartreuse trick worked!

**Comment by wrex** [comment]
> Empty kotoba boxes are unstiled ![image](https://user-images.githubusercontent.com/82216846/212811630-fa442a20-012b-4706-a2df-c6f7c488c07b.png)

Could you post what the markup looks like when this happens? This is tricky for me to reproduce. :smile:



> super minor, but :active buttons in reviews have different background under characters than what the rest of the button uses
> 
> ![image](https://user-images.githubusercontent.com/82216846/212812456-d900a345-eb65-45cd-9553-6885edc17c41.png)

Nice catch! I'd like to separate some of the gray levels a bit more (I already changed one). My old eyes struggle to see the difference.

**Comment by wrex** [comment]
I'm playing with the default colors, trying to stick to your desired aesthetics (as I understand them -- my artistic sensibilities are pretty much nonexistent!). I think I can make it work without introducing any additional hues, but I do need to play with tints/shades and saturation to make everything hold together (with light colored text almost every where and all the semantic significance I'd like to indicate).

Any comments on the colors I'm using below? It makes logical sense to me: 

- The kanji color is red, you're here to review kanji, so the review button is red.
- The SRS stages use the "extra" color (yellow) and gradually get more saturated.
- The lesson button adds things to the apprentice bucket, so it gets the apprentice bucket color.
- The eye is drawn to the review button, not the lesson button which is an explicit goal of mine.

<img width="907" alt="Screenshot 2023-01-17 at 10 17 15 AM" src="https://user-images.githubusercontent.com/653208/212979982-cc8da309-574c-4fe5-9920-bd88b067f3ed.png">

**Comment by wrex** [comment]
I'm unable to reproduce this issue easily (I don't have lessons available that have had collocations added):

> Escapee chartreuse ![image](https://user-images.githubusercontent.com/82216846/212813396-0e547521-d51a-4fe5-bb0e-1933de1d02e9.png)


I think I've fixed it, but there are three issues that I'll need your help with:

1. I'm unsure how light to make the background. I set it to `--ED-surface-3` but it might need to go to `--ED-surface-4`. Could you please check on your end and verify that `-3` looks okay?

2. There are two related rules with `::after` pseudo-elements. I think one is the vertical separator line, but I'm unsure about the other. I've left them chartreuse for now because it's tough for me to write the rules without seeing what I'm doing.

3. The left pointing triangle to the right of the collocation should probably be styled using one of the surface colors. Without being able to reproduce, it's difficult for me to know what to add.

The relevant CSS rules are from line 1402 onward. Here's what it looks like currently:

```css
/* where is this? */
.subject-collocations__patterns::after {
  box-shadow: none;
  right: -9px;
  background-color: chartreuse;
}

.word-types--with-collocations::after {
  box-shadow: none;
  background-color: chartreuse;
  left: calc(100% - 1px);
}

.subject-collocations__pattern-name--selected,
.context__word-types .bg-gray-300,
.context__word-types .bg-gray-300 > span {
  background-color: var(--ED-surface-3);
  color: var(--ED-gray-3);
}
```

**Comment by Sepitus-exe** [comment]
> This one caught me by surprise. I didn't even realize the text stays in that position. I thought it was positioned over the bar. On my screen it's over the yellow. [EDIT: Hmm... This makes me suspect the only easy fix is to either darken the yellow or lighten the gray -- would take serious hacking to figure out which color it is over.]

The bar text is super dynamic, it stays fixed distance away from the right edge of the progress bar on either side depending on where is more space. Its kinda neat 😄 



> It's probably better going forward to open separate issues and assigning them, even for little things like this. Otherwise we risk stepping on each other as we both work on the same things.

I didn't want to overload the issues, but you are right. Thats what they are for after all.

**Comment by Sepitus-exe** [comment]
> Could you post what the markup looks like when this happens? This is tricky for me to reproduce. 😄

Looks like Im too late to the party, but Im glad you figured it out on your own. Hope it wasn't too much of a hassle.

**Comment by Sepitus-exe** [comment]
> Any comments on the colors I'm using below?

The pairing of Lesson/Review buttons with other elements sounds subtle, yet useful. I love the idea.

I'm still not a fan of the A/G/M/E/B colors tho. This is probably going to be a running issue with me. Since they play such I vital role, I'm going to obsess to get them just about right. For now, I would like to try and use the original hues, but get them in line with the R/K/V color feel. I don't know how to better describe it.

**Comment by Sepitus-exe** [comment]
> I'm unable to reproduce this issue easily (I don't have lessons available that have had collocations added):

I will get this one done. If I recall corectly these menu work differently in item pages and reviews, which makes them a pain in the butt to work with. As an example, I beieve one uses the :after pseudo element for the separator, while the other uses a div.

**Comment by wrex** [comment]
Okay, I think I've got the bulk of these resolved (8/10 if I counted correctly).

This one is also hard for me to reproduce at the moment:

https://github.com/Sepitus-exe/WKElementaryDark/issues/23#issuecomment-1384883646

Could you take that one?

Otherwise I think you can accept the PR. 

Feel free to use whatever SRS stage colors work for you as the default. TBH, I'm kinda digging the gradually brighter/saturated single hue (yellow) scheme. It's a subliminal hint above and beyond the words "apprentice/guru/master/enlightened" that fly up as you finish the second reading/meaning quiz for an item. Actually a better UX than with the offical colors, IMO.

**Comment by wrex** [comment]
Crap. Hold on for a bit -- one more commit coming.

Fixing the lessons appears to have introduced a regression with reviews.

**Comment by Sepitus-exe** [comment]
> Could you take that one?

Yeah I will be on it, as well as try my best at some others. There is still that pesky .newbie class using chartreuse that needs to be taken care of among others

**Comment by wrex** [comment]
Okay, I think that's it from me on this issue.

Please accept the PR if it looks like most of these have been addressed, then let's raise separate issues for anything remaining, anything new, or any more regressions introduced.

**Comment by wrex** [comment]
> > Could you take that one?
> 
> Yeah I will be on it, as well as try my best at some others. There is still that pesky .newbie class using chartreuse that needs to be taken care of among others

I haven't a clue where that one might even appear.

**Comment by Sepitus-exe** [comment]
Its on an alert like lement that appears only on lvl one accounts, but I still want to have it covered :)

**Comment by Sepitus-exe** [comment]
I raised some new issues, as well as expanded the amount of info I provide. Feel free to ask for any changes to the way I report the issues as Im failry new to this

**Comment by wrex** [comment]
Fortunately the first three levels are free, so I can create an account for my children to reproduce.

Let's track it in a separate issue once this PR is merged.

**Comment by Sepitus-exe** [comment]
Agreed, I will also be creating some issues for issues Im also currently unable to check for. Mainly charteruse menu in lessons

**Comment by wrex** [comment]
> I raised some new issues, as well as expanded the amount of info I provide. Feel free to ask for any changes to the way I report the issues as Im failry new to this

Me too!

I think you still need to merge my issue23-post-merger-cleanup branch into main, though, before closing this issue. Or did I miss that?

**Comment by Sepitus-exe** [comment]
I did merge your whole branch, didn't I?
